logistics/scripts/modules/LoadTools.psm1

```diff
@@ -221,23 +221,23 @@ function Invoke-SetTestHarnessConfig {
     $testHarnessAppConfig = (Get-ChildItem -Recurse $config.testHarnessAppConfig).FullName
     Write-Host "Editing $testHarnessAppConfig"
 
-    $xml = [xml](Get-Content $testHarnessAppConfig)
+    $jsonFromFile = (Get-Content $testHarnessAppConfig | ConvertFrom-JSON)
 
     if ($config.noExtensions) {
         Write-Host "Disabling Extensions..."
-        if (($key = $xml.SelectSingleNode("//appSettings/add[@key = 'extensions:featureIsEnabled']"))) {
-            $key.SetAttribute("value", "false")
+        if (-not [string]::IsNullOrWhiteSpace($jsonFromFile.ApiSettings.Features[5].IsEnabled)) {
```

Comment:
i think we need to look for the actual key here not not rely on an ordinal position